src/components/SectionRow.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import {IInfographicsRow, IInfographicsSection} from '../types/IInfographicsSection'
+import InfographicsSection from './InfographicsSection'
+
+interface IProps {
+    row: IInfographicsRow,
+    selectedSection: IInfographicsSection | null,
+    handleToggleSelectSection: (section: IInfographicsSection) => void,
+    handleDeleteSection: (section: IInfographicsSection) => void,
+}
+
+const SectionRow = (props: IProps) => {
+    const {row, selectedSection, handleDeleteSection, handleToggleSelectSection} = props
+    const {sections, height} = row
+
+    const generateColumnTemplate = (): string => {
+        let template = ''
```

Comment:
muzes nahradit za array metodu `.reduce()` minimalne doporucuji mrknout co to umi